cin_validator/__main__.py

```diff
@@ -41,7 +41,8 @@ def list_cmd(ruleset):
 @click.option("--output/--no_output", "-o/-no", default=False)
 def run_all(filename: str, ruleset, issue_id, output):
 
-    validator = CinValidationSession(filename, ruleset, issue_id)
+    data_files = cin_class.process_data(filename)
+    validator = cin_class.CinValidationSession(data_files, ruleset, issue_id)
```

Comment:
I think you're right, process date can either go in here, or be part of the process data function itself